doc/getting.rst

```diff
@@ -0,0 +1,65 @@
+Getting
+#######
+
+Requirements
+------------
+
+pyHDLParser requires either Python 2.7 or Python 3.x and no additional libraries.
+
+The installation script depends on setuptools. The source is written in
+Python 2.7 syntax but will convert cleanly to Python 3 when the installer
+passes it through ``2to3``.
+
+Download
+--------
+
+You can access the pyHDLParser Git repository from `Github <https://github.com/hdl/pyHDLParser>`_.
+You can install direct from PyPI with the ``pip`` command if you have it available.
+
+Installation
+------------
+
+pyHDLParser is a Python library.
+You must have Python installed first to use it.
+Most modern Linux distributions and OS/X have it available by default.
+There are a number of options available for Windows.
+If you don't already have a favorite, I recommend getting one of the `"full-stack" Python distros <http://www.scipy.org/install.html>`_ that are geared toward scientific computing such as Anaconda or Python(x,y).
+
+You need to have the Python setuptools installed first.
+If your OS has a package manager, it may be preferable to install setuptools through that tool.
+Otherwise you can use Pip:
+
+.. code-block:: sh
+
+  > pip install setuptools
+
+The easiest way to install pyHDLParser is from `PyPI <https://pypi.python.org/pypi/hdlparse>`_.
```

Comment:
I assume one ticket of order is uploading a new package on PyPI. We should also consider changing the module names from `hdlparse` to `pyhdlparse`.

Reply:
As commented below, I agree. However, we need to decide which namespace to use in PyPI, and modify the Symbolator dependencies accordingly.